runsc/cgroup/cgroup.go

```diff
@@ -214,6 +214,39 @@ func LoadPaths(pid string) (map[string]string, error) {
 	if err := scanner.Err(); err != nil {
 		return nil, err
 	}
+
+	// For nested containers, in /proc/self/cgroup we see paths from host,
+	// which don't exist in container, so recover the container paths here by
+	// double-checking with /proc/pid/mountinfo
+	mountinfo, err := os.Open(filepath.Join("/proc", pid, "mountinfo"))
+	if err != nil {
+		return nil, err
+	}
+	defer mountinfo.Close()
+
+	mfScanner := bufio.NewScanner(mountinfo)
+	for mfScanner.Scan() {
+		txt := mfScanner.Text()
+		fields := strings.Fields(txt)
+		if len(fields) < 9 || fields[len(fields)-3] != "cgroup" {
+			continue
+		}
+		for _, opt := range strings.Split(fields[len(fields)-1], ",") {
+			// we have a subsystem here
```

Comment:
Add the following to handle cgroups with no controller:
```golang
			// Remove prefix for cgroups with no controller, eg. systemd.
			opt = strings.TrimPrefix(opt, "name=")
```

Reply:
Added